src/index.ts

```diff
@@ -610,16 +657,34 @@ export class Bigtable {
    * });
    *
    * //-
+   * // To control how many API requests are made and page through the results
+   * // manually, set `autoPaginate` to `false`.
+   * //-
+   * function callback(err, instances, nextQuery, apiResponse) {
+   *   if (nextQuery) {
+   *     // More results exist.
+   *     bigtable.getInstances(nextQuery, callback);
+   *   }
+   * }
+   *
+   * bigtable.getInstances({
+   *   autoPaginate: false
+   * }, callback);
+   *
+   * //-
    * // If the callback is omitted, we'll return a Promise.
    * //-
    * bigtable.getInstances().then(function(data) {
    *   const instances = data[0];
    * });
    */
-  getInstances(gaxOptions?, callback?) {
-    if (is.function(gaxOptions)) {
-      callback = gaxOptions;
-      gaxOptions = {};
+  getInstances(
+    gaxOptionsOrCallback?: GetInstancesRequest | GetInstancesCallback,
+    callback?: GetInstancesCallback
+  ): void | Promise<GetInstancesResponse> {
+    if (is.function(gaxOptionsOrCallback)) {
```

Comment:
here rather introduce a `gaxOptions` variable (not `gaxOptionsOrCallback`)
Something like
```ts
const gaxOptions = 
  typeof gaxOptionsOrCallback === 'object' ? optionsOrCallback : {};
callback =
  typeof gaxOptionsOrCallback === 'function' ? optionsOrCallback : callback;
```

Reply:
Done!